lua/lvim/bootstrap.lua

```diff
@@ -112,11 +112,15 @@ end
 ---Update LunarVim
 ---pulls the latest changes from github and, resets the startup cache
 function M:update()
-  reload("lvim.utils.hooks").run_pre_update()
-  local ret = reload("lvim.utils.git").update_base_lvim()
-  if ret then
-    reload("lvim.utils.hooks").run_post_update()
-  end
+  require("lvim.core.log"):info "Trying to update LunarVim..."
```

Comment:
```suggestion
```

Reply:
it's basically a duplicate of the line at the top `update_base_lvim()`

https://github.com/LunarVim/LunarVim/blob/325bf72f935aa2caf25cb644f79dc7a3ea2c1168/lua/lvim/utils/git.lua#L61-L62

Reply:
the log in `update_base_lvim` shows up only after the command finishes